sdk/src/testIntegration/java/com/hedera/hashgraph/sdk/test/integration/ContractFunctionParametersIntegrationTest.java

```diff
@@ -2788,6 +2788,39 @@ void canCallContractFunctionBytesArrayType() throws Exception {
         assertThat(responseResult).isEqualTo(testBytes);
     }
 
+
+    @Test
+    @DisplayName("Can receive bytes4 value from contract call")
+    void canCallContractFunctionBytes4Type() throws Exception {
```

Comment:
I suggest removing the integration tests, since they will be failing because the contract we are using is missing the function `returnBytes4` and it is outside the scope of this task. 
Please create a follow up issue when this PR is merged to implement these integration tests

Reply:
Removed the integration tests